client/my-sites/plan-features/header.jsx

```diff
@@ -51,13 +51,15 @@ class PlanFeaturesHeader extends Component {
 	}
 
 	renderPlansHeader() {
-		const { planType, popular, newPlan, title, translate } = this.props;
+		const { newPlan, planType, popular, selectedPlan, title, translate } = this.props;
 
 		const headerClasses = classNames( 'plan-features__header', getPlanClass( planType ) );
+
 		return (
 			<header className={ headerClasses } onClick={ this.props.onClick }>
-				{ popular && <Ribbon>{ translate( 'Popular' ) }</Ribbon> }
-				{ newPlan && <Ribbon>{ translate( 'New' ) }</Ribbon> }
+				{ planType === selectedPlan && <Ribbon>{ translate( 'Suggested' ) }</Ribbon> }
+				{ popular && ! selectedPlan && <Ribbon>{ translate( 'Popular' ) }</Ribbon> }
+				{ newPlan && ! selectedPlan && <Ribbon>{ translate( 'New' ) }</Ribbon> }
```

Comment:
As that ribbon logic is growing, it seems like it might be useful to move it in a separate component. That applies for a bunch of stuff here in the plans features (like prices for example), and can be addressed in separate PRs.

Reply:
Yes, makes sense! For the sake of not stalling this improvement, I'd prefer it to be handle in a separate PR.